snapfaas/src/vm.rs

```diff
@@ -107,7 +106,7 @@ struct VmHandle {
     // killed, before the VmHandle is dropped.
     vm_process: Child,
     // None when VM is created from single-VM launcher
-    invoke_handle: Option<Sender<Message>>,
+    invoke_handle: Option<Arc<Mutex<Scheduler>>>
```

Comment:
I think a better way is to not have `invoke_handle` as part of VmHandle but pass it a function argument.

The RPC connection is really _not_ a VM property. Instead, it should be a worker thread property. Any VM should use the connection of whichever thread it is on.

Reply:
Sounds reasonable. Fixed